src/main/java/mate/academy/service/UserService.java

```diff
@@ -3,18 +3,25 @@
 import mate.academy.model.User;
 
 public class UserService {
-    private static final User[] users = new User[] {
```

Comment:
I am almost sure that this declaration was according to the style convention, so please double-check this moment

Reply:
What I meant here was about the space between `[]` and `{`, but since our `users` array is effectively final and constant here - we can name it like you did, anyway this is not that important, so you can leave it as it is :)